.github/workflows/archive-full-index.yaml

```
@@ -0,0 +1,121 @@
+name: Generate the full index archive for Bitnami Charts
+on:
+  push:
+    branches:
+      - index
+jobs:
+  get:
+    runs-on: ubuntu-latest
+    name: Get
+    steps:
+      - id: checkout-repo-index
+        name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          ref: index
+          path: index
+      - id: checkout-repo-full-index
+        name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          ref: full-index
+          path: full-index
```

This is not the correct branch name, right?

```suggestion
          ref: archive-full-index
          path: full-index
```